app/views/members/users/setup.html.haml

```diff
@@ -25,6 +19,10 @@
   If you later need to update your Google Drive/Calendar email address, please email
```

Comment:
It looks like some of the intermediate edits to that reference page didn't get flagged. Can you change this paragraph to say

If don’t want to use the Google-friendly email address you entered above for the DU mailing list, or you later need to update your Drive/Calendar/mailing list email address, please let us know at membership@doubleunion.org and we’ll get you sorted!